examples/uppy-with-companion/server/index.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express')
 const bodyParser = require('body-parser')
 const session = require('express-session')
-const companion = require('../../../packages/@uppy/companion/src/companion')
+const companion = require('@uppy/companion/src/companion')
```

Comment:
```suggestion
const companion = require('@uppy/companion')
```

Reply:
I tried `require('@uppy/companion')` but again raise an import error that can't load.

Reply:
That's probably because you need to run `corepack yarn build` first – we should document that BTW.